src/lib/components/progressbar/ProgressBar.svelte

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+    import type { ProgressbarData, ProgressbarProps } from '$lib/components';
+
+    type $$Props = ProgressbarProps;
+
+    /**
+     * The max value of the progressbar
+     */
+    export let maxSize: $$Props['maxSize'];
+
+    /**
+     * The remaining value of the progressbar
+     */
+    let remainder = $$props.data.reduce(
+        (sum: number, item: ProgressbarData) => sum - item.size,
+        maxSize
+    );
+</script>
+
+<section class="progressbar__container">
+    {#each $$props.data as item}
+        <div
+            class="progressbar__content"
+            style="background-color:{item.color}; width: {(item.size / maxSize) * 100}%;">
```

Comment:
```suggestion
            style:background-color={item.color}
            style:width={`${(item.size / maxSize) * 100}%`}>
```
the short hands will be cleaner here 👌🏻